### DISCUSSION TITLE
[new feature request] Strong need in commenting on commit changes!!!

### DISCUSSION THREAD

**Discussion by planetb113** [ReviewNB/support#111] [Ideas]
![image](https://user-images.githubusercontent.com/33831002/178109767-8b8fed91-642b-43b5-be4e-b259f9b59d3d.png)
![image](https://user-images.githubusercontent.com/33831002/178109841-82df3824-4d26-4ded-acc4-e80944417906.png)
![Screen Shot 2022-07-09 at 10 21 56 PM](https://user-images.githubusercontent.com/33831002/178109966-7bbef956-5459-4588-a9f1-c4f7ef084c06.png)

**Comment by amit1rrr** [comment]
We've discussed this before here - [https://github.com/ReviewNB/support/issues/93](https://github.com/ReviewNB/support/issues/93)

Quoting from it -

> Thanks for the explanation.
> 
> By design, we only allow 2 type of comments -
> 
> 1. Pull Request Comments - Typical software development workflow wherein a set of changes are reviewed until they're approved for merge.
> 2. JDoc comments - GoogleDoc style comments on notebook itself. These are good for a quick comment or discussion with non-technical stakeholders who typically don't care about the diff view.
> 
> We believe these 2 types of notebook comments cover all the collaboration requirements of data science teams. Please let me know if there's any workflow that doesn't fit either of these.
> 
> Now here's the main issue with Commit level comments -
> 
> * Lack of container for continued discussion. E.g. Say an author receives some feedback on their work and they make some changes in response to that feedback. Now the new work is in a separate commit. What if reviewers have some more feedback on the updated notebook? They need to open a new discussion on the new commit. That way a single discussion is spread over multiple commits and it's hard to track all open discussions. Pull Request solves this exact problem.
